src/global/tracking/tracking.cc

```diff
@@ -57,7 +58,13 @@ void InitPlugin(JApplication* app) {
       app));
 
   app->Add(new JOmniFactoryGeneratorT<TrackerMeasurementFromHits_factory>(
-      "CentralTrackerMeasurements", {"CentralTrackingRecHits"}, {"CentralTrackerMeasurements"},
+      "CentralTrackerMeasurements", {"CentralTrackingRecHits"}, {"CentralTrackerRecMeasurements"},
```

Comment:
Could we be more explicit in the naming here, something like:
```suggestion
      "CentralTrackerNoTOFBarrelMeasurements", {"CentralTrackingRecHits"}, {"CentralTrackerNoTOFBarrelMeasurements"},
```
This collection should also be added to the output file.